checkins/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
```

Comment:
See https://github.com/Larissa-Developers/prizy_backend/wiki for the API endpoints.

The mobile client is following this wiki as well